### ISSUE TITLE
在执行test_inference.py时出现异常

### ISSUE THREAD

**Issue by syvince** [fengin/Fun-ASR-Nano-2512-Deploy#8]
在执行test_inference.py时出现如下异常：
`=== Fun-ASR 推理测试 ===
Initial VRAM: Allocated: 0.00 GB, Reserved: 0.00 GB
Running on device: cuda
Loading model from local path: /home/vsislab/funasr/Fun-ASR/Fun-ASR-Nano-2512-Deploy/models/FunAudioLLM/Fun-ASR-Nano-2512
FPS16: False
funasr version: 1.2.9.
Check update of funasr, and it would cost few times. You may disable it by set `disable_update=True` in AutoModel
You are using the latest version of funasr-1.2.9
WARNING:root:trust_remote_code: True
Loading remote code failed: model, No module named 'model'
Model loaded. VRAM: Allocated: 1.99 GB, Reserved: 2.01 GB

Transcribing: https://isv-data.oss-cn-hangzhou.aliyuncs.com/ics/MaaS/ASR/test_audio/asr_example_zh.wav
Audio bytes len: 177572

=== Testing with raw bytes input (Should Fail) ===
  0%|                                                                                          | 0/1 [00:00<?, ?it/s]Failed with bytes: list index out of range
  0%|                                                                                          | 0/1 [00:00<?, ?it/s]

=== Testing with decoded numpy array (Solution) ===
Decoded shape: (88747,), dtype: float32
Input: [audio_data]
  0%|                                                                                          | 0/1 [00:00<?, ?it/s]The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
rtf_avg: 1.272: 100%|██████████████████████████████████████████████████████████████████| 1/1 [00:07<00:00,  7.10s/it]
[{'key': 'rand_key_1t9EwL56nGisi', 'text': '(norm************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴SleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleep', 'text_tn': 'norm폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴폴SleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleepSleep', 'label': 'null'}]

=== Recognition Result (BYTES TEST) ===
Final VRAM: Allocated: 3.11 GB, Reserved: 3.20 GB`

**Comment by fengin** [comment]
Loading remote code failed: model, No module named 'model'——你的模型依赖文件有下载全么? 检查下安装的funasr-1.2.9有没下载全

**Comment by fengin** [comment]
另外，建议升级到1.3.0，这个版本支持热词了，我本地升级了，这里开源的没更新

**Comment by syvince** [comment]
非常感谢，我解决了，模型好像是没有完全加载正确
